src/Tribe/Attendee_Info.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * Class Tribe__Tickets__Attendee_Info
+ */
+class Tribe__Tickets__Attendee_Info {
+
+	public function hook() {
```

Comment:
Ideally this hooking should not be here but in the service provider; unless absolutely required we're trying to keep any filtering in the service providers and remove it from the classes to avoid side-effects.
Examples of new and previous code might be confusing but that's the general idea.